Fix the Nuttx build on Travis CI

The following commit in NuttX tools cause a build failure on the Travis CI: https://bitbucket.org/nuttx/tools/commits/164450f982b404fdc2b3233db51dc3eaa1f08b7f

**output:**
```
make -C ../tools/kconfig-frontends
make[1]: Entering directory `/home/travis/build/jerryscript-project/tools/kconfig-frontends'
/home/travis/build/jerryscript-project/tools/kconfig-frontends/scripts/.autostuff/scripts/missing: line 81: aclocal-1.15: command not found
WARNING: 'aclocal-1.15' is missing on your system.
         You should only need it if you modified 'acinclude.m4' or
         'configure.ac' or m4 files included by 'configure.ac'.
         The 'aclocal' program is part of the GNU Automake package:
         <http://www.gnu.org/software/automake>
         It also requires GNU Autoconf, GNU m4 and Perl in order to run:
         <http://www.gnu.org/software/autoconf>
         <http://www.gnu.org/software/m4/>
         <http://www.perl.org/>
make[1]: *** [aclocal.m4] Error 127
make[1]: Leaving directory `/home/travis/build/jerryscript-project/tools/kconfig-frontends'
make: *** [install-kconfig] Error 2
```

--- a/targets/nuttx-stm32f4/Makefile.travis
+++ b/targets/nuttx-stm32f4/Makefile.travis
@@ -31,7 +31,9 @@ LOCAL_INSTALL:=$(CURDIR)/../local/
 install-kconfig:
 	git clone https://bitbucket.org/nuttx/tools.git ../tools
 	mkdir -p $(LOCAL_INSTALL)
-	cd ../tools/kconfig-frontends && ./configure --disable-mconf --disable-nconf --disable-gconf --disable-qconf --disable-utils --disable-shared --enable-static --prefix=$(LOCAL_INSTALL)
+	# FIXME: 'autoreconf --force --install' is a workaround after
+	#        https://bitbucket.org/nuttx/tools/commits/164450f982b404fdc2b3233db51dc3eaa1f08b7f
+	cd ../tools/kconfig-frontends && autoreconf --force --install && ./configure --disable-mconf --disable-nconf --disable-gconf --disable-qconf --disable-utils --disable-shared --enable-static --prefix=$(LOCAL_INSTALL)
 	$(MAKE) -C ../tools/kconfig-frontends
 	$(MAKE) -C ../tools/kconfig-frontends install
 

